Gemfile

```diff
@@ -52,3 +52,7 @@ group :test do
 end
 
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+
+# Explicitly depend on and compile nokogiri
+# so we can run CI on Ruby head
+gem 'nokogiri', force_ruby_platform: true
```

Comment:
Hi, I was wondering if we could add a pessimistic version for this gem? As a user of the gem, I often find that using a pessimistic version can be helpful when dealing with version dependencies. It would be great if we could include this option for other users who may find it useful as well. Thank you!